docs/paper/paper.md

```diff
@@ -98,6 +98,11 @@ temperature and perfect reflectors:
 
 The implementation for the plane-cylinder geometry is based on [@emig_prl_2006].
 
+Other packages that allow to compute the Casimir free energy are ``SCUFF-EM``
+[@reid_ieee_2015] and ``Meep`` [@oskooi_cpc_2010].  While both packages support
+arbitrary geometries and ``SCUFF-EM`` also has support for non-equilibrium
+Casimir forces, they are limited too much smaller aspect ratios than ``CaPS``.
```

Comment:
"too" -> "to"

Reply:
It might be appropriate to add one sentence to make the difference between these packages and ``CaPS`` more obvious. The following suggestion might not be the optimum yet but should illustrate what I have in mind:

Both packages support arbitrary geometries and ``SCUFF-EM`` also has support for non-equilibrium Casimir forces. In contrast, ``CaPS`` specializes to the plane-sphere geometry where it allows to cover the aspect ratios of the vast majority of existing experiments.